### PR TITLE
Add missing dependency information

### PR DESCRIPTION
**Pull request by ambihelical**
The following additional packages were needed to build:
libxml2-utils, xsltproc

### CODE REVIEW

**Comment by stapelberg** [comment]
On which distribution is that? We’re not using these libraries directly in the i3status build process, so asciidoc should pull them in.

**Comment by ambihelical** [comment]
Mint 17.1

**Comment by ambihelical** [comment]
Looking into it a little further, libxml2-utils is a recommended package, not a dependency.  xsltproc is a dependency of at least one recommended package as well.   My system setup scripts don't install recommended packages, so I guess this is technically a problem with my setup.

**Comment by Airblader** [comment]
If a build fails due to a missing package, that package should be a hard dependency, not a recommendation. Michael's point is just that these two dependencies aren't dependencies of i3 itself, but of asciidoc which in turn is a dependency of i3.

**Comment by ambihelical** [comment]
Just to be clear, I was refering to asciidoc's recommended packages in my previous comment.   Asciidoc is not one of the packages I install with `apt-get install --no-install-recommends`, but it may have been pulled in as a dependency by some other package using that option. I can't really say at this point, but it would explain why those packages were missing.

**Comment by stapelberg** [comment]
Yeah, that’s what I suspected.

https://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps defines:

> Recommends
> This declares a strong, but not absolute, dependency.
> The Recommends field should list packages that would be found together with this one in all but unusual installations.

The way I interpret this is that users should not disable Recommends unless they are willing to accept situations like the one you found yourself in :).

Unfortunately, adding these as dependencies doesn’t solve the issue at hand in a reliable way: it could be that asciidoc gains another Recommends in future versions, and then we’d need to play catch-up. Also, asciidoc might drop the Recommends at some point, and then we are pulling in an unnecessary dependency.

In conclusion, I don’t think adding these dependencies is a good idea.
